src/kitovu/sync/plugin/smb.py

```diff
@@ -135,14 +144,24 @@ def create_local_digest(self, path: pathlib.Path) -> str:
         return self._create_digest(size=info.st_size, mtime=info.st_mtime)
 
     def create_remote_digest(self, path: pathlib.PurePath) -> str:
-        attributes = self._connection.getAttributes(self._info.share, str(path))
+        try:
+            attributes = self._connection.getAttributes(self._info.share, str(path))
+        except OperationFailure:
+            raise utils.PluginOperationError(
+                f'Could not find remote file {path} in share "{self._info.share}"')
```

Comment:
Not sure if we should handle this - this is something which should never happen, right?

Reply:
If you're very unlucky it could happen. A file could disappear directly after listen the files (the same goes for sub directories). I'd prefer to check it and print something understandable rather than breaking the whole application.